File: Images/app/src/main/java/rob/myappcompany/images/MainActivity.java
package rob.myappcompany.images;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {

    public void switchImage(View view){

        ImageView image = (ImageView) findViewById(R.id.rightImageView);
        image.setImageResource(R.drawable.left);
    }


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: JavaFaithful/app/src/main/java/rob/myappcompany/javafaithful/MainActivity.kt
package rob.myappcompany.javafaithful

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import rob.myappcompany.javafaithful.R
import rob.myappcompany.javafaithful.Dog

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val myDog = Dog("Tommy", 5)
    }
}